doc/source/dev/roadmap-detailed.rst

```diff
@@ -340,8 +347,9 @@ isolve:
 dsolve:
 
 - add sparse Cholesky or incomplete Cholesky
-- look at CHOLMOD
-
+- add sparse QR
+- improve interface to SuiteSparse UMFPACK
```

Comment:
Currently, I don't think there's any confirmation to users whether UMFPACK is actually being used by `scipy.sparse.linalg.spsolve`. This should be more clear. There should probably also be ways of just doing LU decomposition with UMFPACK (e.g. work into `scipy.sparse.linalg.splu`).

Reply:
That sounds useful - more an issue to be filed than a roadmap item probably.